ci/README.md

```diff
@@ -1,8 +1,10 @@
-docker run -it --entrypoint=/bin/bash -v $PWD/skyhook/ci:/ci -v $PWD/skyhookdb-ceph:/ceph ceph/daemon:v3.0.7-stable-3.0-kraken-ubuntu-16.04-x86_64
-./micro-osd deploy
-docker run -it --rm --name test -v $PWD/skyhookdb-ceph:/ceph cephbuilder/ceph:kraken
-make run-query
+# CI workflows
 
-TODO: pull build container and compile run-query and cls, pull ceph/daemon:kraken + raw data in container, run micro-osd, and run containers
- package raw data into container, pull container
+The [smoke test workflow](workflows/smoke.workflow) builds the skyhook 
+fork and RADOS class, and subsequently executes the test (upload, 
+index and query data). To run it:
 
+```bash
+cd skyhook-ceph/
+popper run --recursive
```

Comment:
can you please add to readme that this is how to run the tests offline, without travis, only dependency is popper and docker is installed locally.

Reply:
I second this request.